ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```
@@ -700,6 +701,15 @@ keys:
                 Only applicable if `source_protocol` is one of `connected`, `static`, `isis`, `user`, `dynamic`.
             include_leaked:
               type: bool
+            ospf_route_type:
+              type: str
+              valid_values:
+                - "external"
+                - "internal"
+                - "nssa-external"
+                - "nssa-external 1"
+                - "nssa-external 2"
+              description: The `ospf_route_type` is required with source_protocols 'ospf' and 'ospfv3'.
```

i dont this required word is correct here. We can have redistribute 'ospf' and 'ospfv3' without ospf_route_type. 
```suggestion
              description: The `ospf_route_type` is valid with source_protocols 'ospf' and 'ospfv3'.
```

---

Made it same for all like the below comment.